src/data-workspace/display/display.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { CustomDataProvider } from '@dhis2/app-runtime'
+import { render, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react'
```

Comment:
I used `@testing-library/react` instead of `enzyme` for these tests as the various utils provided such as `waitForElementToBeRemoved` and `getByRole` made the tests much clearer.